internal/logger/logger_test.go

```diff
@@ -0,0 +1,58 @@
+package logger
+
+import (
+	"github.com/sirupsen/logrus"
```

Comment:
imports should always be formatted as 
```
import (
        "some/builtin/package"
        "some/builtin/package2"

        "github.com/some/external/package"

        "github.com/osbuild/image-builder/internal/ourownstuff"
)

Reply:
Prepare source says otherwise https://github.com/osbuild/image-builder/pull/73/checks?check_run_id=1537038729. I will also try with a blank line in between but I don't think that matters.

Reply:
OK, I was wrong. The blank lines do matter. Didn't know that.

Reply:
Yea it's not checked in prepare-source. It's just a convention I guess.